tvbwidgets/api.py

```diff
@@ -5,6 +5,8 @@
 # (c) 2022-2023, TVB Widgets Team
 #
 
+from .ui.connectivity_react.connectivity_widget import ConnectivityWidgetReact
```

Comment:
Should we even put this connectivity_react module in the API, considering that we do not consider it production ready?
With the import here, it will oblige us to import its dependencies immediately, even if we do not use (or intend to use this widget).
I would recommend that we keep the code, but not list it in the API, yet

Reply:
Yes, I agree with your observation. I have removed the import from the latest version and also update the commented usage of said widget.